internal/pkg/template/input/input_test.go

```diff
@@ -36,106 +36,135 @@ func TestTemplateInputsValidateDefinitions(t *testing.T) {
 	assert.Error(t, err)
 	assert.Contains(t, err.Error(), `key="type"`)
 
-	// Fail - defined options for wrong Kind
+	// Fail - input Kind with missing Type
```

Comment:
Hodil jsem testy týkající se selectů do samostatné metody níž, tak se to tu jen přeskládalo.